parts/windows/windowscontainerdfunc.ps1

```diff
@@ -126,7 +126,7 @@ function Enable-Logging {
     $logs = Join-path $pwd.drive.Root logs
     Write-Log "Containerd hyperv logging enabled; temp location $logs"
     $diag = Join-Path $global:ContainerdInstallLocation diag.ps1
-    mkdir -Force $logs

```

Comment:
Does `mkdir -Force` equal our new function `Create-Directory`? because it will create the folder anyway.

Reply:
mkdir -Force can do the same thing. Just keep them in a same format with using a same function.

Reply:
And share the same code and format for log.

Reply:
mkdir is a function which calls New-Item to create directories specifically. I prefer to use New-Item in PowerShell script